packages/commons/src/pet.class.ts

```diff
@@ -0,0 +1,16 @@
+import { v4 } from 'uuid';
+
+export default class Pet {
+  id: string;
```

Comment:
This is not used and a duplicate of the internal `_id` so I would ask you to remove it.

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317